package.json

```diff
@@ -142,33 +120,27 @@
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "yarn test:deprecation-policy",
     "prepack": "sf-prepack",
-    "prepare": "yarn compile && yarn lint",
+    "prepare": "sf-install",
```

Comment:
Do these do the same thing?

Reply:
It doesn't seem like it based on [sf-install.js](https://github.com/forcedotcom/dev-scripts/blob/main/bin/sf-install.js). Why do we want to change this behavior?

Reply:
Ah I guess not